pom.xml

```diff
@@ -89,6 +88,7 @@
         <module>m_Titles</module>
         <module>m_Turrets</module>
         <module>m_Vault</module>
+        <module>m_Duel</module>
```

Comment:
Can you add a comment like "Duel needs Vault dependency"? Just do avoid a future mistake...